appveyor.yml

```diff
@@ -1,18 +1,26 @@
 version: '{build}'
 skip_tags: true
-image: Visual Studio 2017
+image:
+- Visual Studio 2019
+- Ubuntu1804
```

Comment:
Just change to `Ubuntu`, I think `Ubuntu1804` doesn't have anything preinstalled, see [linux-images-software](https://www.appveyor.com/docs/linux-images-software/#operating-system).
And I beleive after that change you don't need setup.sh since CLI will be preinstalled

Reply:
@skomis-mm Changed to image Ubuntu and did different tests with and without setup.sh, but no change in the error message about the test process

Reply:
@ericvb, almost successfull was the [test completely with no install step](https://github.com/serilog/serilog-sinks-email/pull/81#discussion_r424906466).
The only problem was specified version  `3.1.202` in global.json which is higher than [installed](https://www.appveyor.com/docs/linux-images-software/#operating-system). Try to use the lowest possible version with `latestFeature` roll-forward policy, like [here](https://github.com/serilog/serilog-sinks-email/pull/81#discussion_r424906466), for example.

Reply:
@skomis-mm, thanks, learned about some new parameters in the global.json file and the used appveyor build system